app/views/games/index.html.erb

```diff
@@ -1 +1,14 @@
-<h1>"Here's the games#index view</h1>
\ No newline at end of file
+<div class="col-10 offset-1">
+  <h1>"Here's the games#index view"</h1>
+
+  <% @games.each do |game| %> 
```

Comment:
We will want to show just the list of available games to join so let's utilize the **@pending_games** variable ( _or whatever name you choose for the available games instance variable_ ) and iterate over those instead of @games